opentimelineio/adapters/cmx_3600.py

```diff
@@ -544,8 +544,9 @@ def expand_transitions(timeline):
     return timeline
 
 
-def read_from_string(input_str):
-    parser = EDLParser(input_str)
+def read_from_string(input_str, **edl_args):
+    rate = edl_args.get('rate', 24)
```

Comment:
I think you could use `rate=24` instead of `**edl_args` here. I don't anticipate many more arguments, so it should be fine.

Reply:
I'll change it. Thanks.